executable-spec/src/Cardano/Ledger/Spec/STS/Update/Tallysip.hs

```diff
@@ -184,7 +184,7 @@ tallyOutcome vres stakeDistribution pNoQ pNoM r_a =
         else
           if Data.stakePercentRound (Data.stakeAbstain vres) (totalStake stakeDistribution)
              > vThreshold r_a
-             && Data.rvNoQuorum vres <= pNoQ
+             && Data.rvNoQuorum vres < pNoQ
```

Comment:
I'm curious, did you change `<=` to `<` as a result of failing tests?

Reply:
Not a "<=" so as not to run a revote pNoQ+1 times

I have put this in a comment